src/quantity-selector/index.ts

```diff
@@ -0,0 +1,56 @@
+import { alias, tag, Events, Tag } from '@storefront/core';
+
+const DEFAULT_VALUE = 1;
+
+@alias('quantitySelector')
```

Comment:
Does it need to alias itself if it doesn't allow nesting? (there's no yield or child tags)